# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/Key.h
# Commit: 9966f5d71bb2
# Full Hash: 9966f5d71bb2db2228a914743dca457dcc337d02
# Author: Simon Giesecke <sgiesecke@mozilla.com>
# Date: 2019-09-18 21:49:27
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Apply renamings suggested for Cursor::*Key members and related identifiers. r=ytausky,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D43249
# ==============================================================================

diff -r 714d36c43204 -r 9966f5d71bb2 dom/indexedDB/Key.h
--- a/dom/indexedDB/Key.h	Wed Sep 18 06:41:23 2019 +0000
+++ b/dom/indexedDB/Key.h	Wed Sep 18 06:41:35 2019 +0000
@@ -159,7 +159,7 @@
       JSContext* aCx, bool aFirstOfArray, JS::Handle<JS::Value> aVal,
       ErrorResult& aRv);
 
-  IDBResult<void, IDBSpecialValue::Invalid> ToLocaleBasedKey(
+  IDBResult<void, IDBSpecialValue::Invalid> ToLocaleAwareKey(
       Key& aTarget, const nsCString& aLocale, ErrorResult& aRv) const;
 
   void FinishArray() { TrimBuffer(); }
